front/components/data_source_view/DataSourceViewSelector.tsx

```diff
@@ -39,7 +39,7 @@ const MIN_DATA_SOURCES_PER_KIND_TO_GROUP = 3;
 const ONLY_ONE_VAULT_PER_SELECTION = true;
 
 interface DataSourceViewsSelectorProps {
-  owner: LightWorkspaceType;
+  owner: WorkspaceType;
```

Comment:
Why all the LightWorkspaceType => WorkspaceType change ? (usually, the leaf component do not _really_ need the full type)

Reply:
I need to pass the "flags" to the `OptionsComponent` within the `ConnectorPermissionModal` which are not present in the `LightWorkspaceType`.